src/github/taskDashboardWebviewProvider.ts

```diff
@@ -0,0 +1,1180 @@
+				const issueNumber = parseInt(issueNumberStr, 10);
+
+				for (const folderManager of this._repositoriesManager.folderManagers) {
+					const issueModel = await folderManager.resolveIssue(owner, repo, issueNumber);
+					if (issueModel) {
+						// Use the extension's command to open the issue description
+						await vscode.commands.executeCommand('issue.openDescription', issueModel);
+						return;
+					}
+				}
+			}
+
+			// Fallback to opening externally if we can't find the issue locally
+			await vscode.env.openExternal(vscode.Uri.parse(issueUrl));
+		} catch (error) {
+			Logger.error(`Failed to open issue: ${error} `, DashboardWebviewProvider.ID);
+			// Fallback to opening externally
+			try {
+				await vscode.env.openExternal(vscode.Uri.parse(issueUrl));
+			} catch (fallbackError) {
+				vscode.window.showErrorMessage('Failed to open issue.');
+			}
+		}
+	}
+
+	private async openPullRequest(pullRequest: { number: number; title: string; url: string }): Promise<void> {
+		if (!pullRequest) {
+			return;
+		}
+
+		try {
+			// Try to find the pull request in the current repositories
+			for (const folderManager of this._repositoriesManager.folderManagers) {
+				// Parse the URL to get owner and repo
+				const urlMatch = pullRequest.url.match(/github\.com\/([^\/]+)\/([^\/]+)\/pull\/(\d+)/);
+				if (urlMatch) {
+					const [, owner, repo] = urlMatch;
+					const pullRequestModel = await folderManager.resolvePullRequest(owner, repo, pullRequest.number);
+					if (pullRequestModel) {
+						// Use the extension's command to open the pull request
+						await vscode.commands.executeCommand('pr.openDescription', pullRequestModel);
+						return;
+					}
+				}
+			}
+
+			// Fallback to opening externally if we can't find the PR locally
+			await vscode.env.openExternal(vscode.Uri.parse(pullRequest.url));
+		} catch (error) {
+			Logger.error(`Failed to open pull request: ${error} `, DashboardWebviewProvider.ID);
+			// Fallback to opening externally
+			try {
+				await vscode.env.openExternal(vscode.Uri.parse(pullRequest.url));
+			} catch (fallbackError) {
+				vscode.window.showErrorMessage('Failed to open pull request.');
+			}
+		}
+	}
+
+	private async startRemoteAgent(issueData: any): Promise<void> {
+		if (!issueData || !issueData.url) {
+			return;
+		}
+
+		try {
+			// Parse the issue URL to get owner, repo, and issue number
+			const urlMatch = issueData.url.match(/github\.com\/([^\/]+)\/([^\/]+)\/issues\/(\d+)/);
+			if (urlMatch) {
+				const [, owner, repo, issueNumberStr] = urlMatch;
+				const issueNumber = parseInt(issueNumberStr, 10);
+
+				// Find the folder manager for this repository
+				for (const folderManager of this._repositoriesManager.folderManagers) {
+					const issueModel = await folderManager.resolveIssue(owner, repo, issueNumber);
+					if (issueModel) {
+						// Use the new side-by-side command
+						await vscode.commands.executeCommand('issue.openIssueAndCodingAgentSideBySide', issueModel);
```

Comment:
The existing `createOrShow` methods have a `toTheSide` option, no need to for a new command.

https://github.com/microsoft/vscode-pull-request-github/blob/e5ce10696b0010b836f8a9a96a4ecd6a33295a48/src/github/issueOverview.ts#L39-L46